metacat-user-metadata-mysql/src/main/java/com/netflix/metacat/usermetadata/mysql/MySqlUserMetadataConfig.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.usermetadata.mysql;
+
+import com.netflix.metacat.common.json.MetacatJson;
+import com.netflix.metacat.common.server.properties.Config;
+import com.netflix.metacat.common.server.usermetadata.UserMetadataService;
+import com.netflix.metacat.common.server.util.DataSourceManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * MySql UserMetadata Config.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+@Configuration
+@Profile({"dev", "test", "prod"})
```

Comment:
Instead of all these profiles I would just create a profile like "mysql-metadata". We shouldn't have to have all these just to enable this feature. dev, test, prod should probably moreso be reserved for swapping out properties like database connection strings or catalog locations.